azurerm/resource_arm_cosmosdb_mongo_database.go

```diff
@@ -49,11 +52,17 @@ func resourceArmCosmosDbMongoDatabase() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validate.CosmosAccountName,
 			},
+
+			"throughput": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ValidateFunc: validate.CosmosThroughput,
+			},
 		},
 	}
 }
 
-func resourceArmCosmosDbMongoDatabaseCreate(d *schema.ResourceData, meta interface{}) error {
+func resourceArmCosmosDbMongoDatabaseCreateUpdate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Could we use separate create and update functions? i think it'll simplify things a tad